openassessment/tests/test_data.py

```diff
@@ -1317,7 +1317,110 @@ def test_create_zip_with_attachments(self):
             self.answer_text.encode('utf-8')
         )
 
-        self.assertTrue(zip_file.read(os.path.join(COURSE_ID, 'downloads.csv')))
```

Comment:
oops, i don't think we want to delete this line from this other test, I think it's still worthwhile to assert that the file exists